### ISSUE TITLE
How to set account avatar?

### ISSUE THREAD

**Issue by zoujiaqing** [sebadob/rauthy#558]
Use minio or s3 as avator storage?

**Comment by sebadob** [comment]
Rauthy does not implement any "social" stuff on purpose currently.

**Comment by zoujiaqing** [comment]
I think the basic information of users is necessary. This is not a social attribute. For example, if I login with github authorization, I will also get an avatar.

**Comment by sebadob** [comment]
Right, basic info is necessary, but I don't think of an avatar as basic info. This is a gimmick.
This might come in the future if no other more important things need to be done, but not at the moment.

The whole persistence layer is about to be migrated and the UI will get a rework afterward. At some point in the future this may come.

**Comment by sebadob** [comment]
I have 2 big things on my TODO, but after these are done, I will most probably add an avatar option to Rauthy.

**Comment by sebadob** [comment]
User Avatars will be included in the upcoming releease.
